### ISSUE TITLE
Auto-generated header/footer

### ISSUE THREAD

**Issue by vcc-LG** [adamburmister/gitprint.com#55]
Do you think it would be possible to add in a feature whereby the file's title, last commit date, today's date, author etc. could be added into the header/footer of the printout?

**Comment by adamburmister** [comment]
Yeah it would be possible but would kill the caching (and hence speed). I'll keep this suggestion around for next version.
